tests/test_benchmark.py

```diff
@@ -354,7 +357,7 @@ def setUpClass(self):
 
   # avoid depending on argument reception from the commandline
   def hardcode_arguments(self, code):
-    if not code:
+    if not code or 'int main()' in code:
```

Comment:
this looks bad - we need to hardcode that argument. where is it not possible to do so?

Reply:
Do we need to hardcode that argument? If the source code uses `int main()`, it doesn't care what the arguments are.

Reply:
It would be nice if all benchmarks followed that convention, but yeah, I guess some of the non-core ones don't?